penrose-web/src/inspector/views/mod/LabeledInput.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from "react";
-import {toHex} from "../../../Util";
+import {toHex} from "../../../utils/Util"
```

Comment:
BTW I changed the module resolution rules and you can just use absolute paths! `utils/Util` would work just fine.